products/mqtt-broker/mqtt-broker.sass

```diff
@@ -117,17 +117,14 @@
         display: block
         height: 18px
         width: 18px
-        mask: url('/images/trendz/arrow-right.svg') no-repeat 50% 50%
-        -webkit-mask: url('/images/trendz/arrow-right.svg') no-repeat 50% 50%
-        mask-size: cover
-        -webkit-mask-size: cover
-        background-color: #fff
+        background-image: url('/images/trendz/arrow-right.svg')
```

Comment:
Why the image is trendz?

Reply:
To avoid uploading the same image (arrow), we decided to use the existing one (that is used in trendz prod page) and add rotation + style.